budcamp/Homework1/Homework1/Program.cs

```diff
@@ -0,0 +1,281 @@
+
+            Task8();
+
+            Task9();
+
+            Task10();
+
+            Task11();
+
+            Task12();
+
+            Task13();
+        }
+
+        public static void Task1()
+        {
+            int number = 24;
+            Console.WriteLine($"variable name is: {nameof(number)} and its value is: {number}");
+        }
+
+        private static void Task2()
+        {
+            Console.WriteLine("enter first num");
+            double input1 = int.Parse(Console.ReadLine());
+            Console.WriteLine("enter second num");
+            double input2 = int.Parse(Console.ReadLine());
+            Console.WriteLine("enter third num");
+            double input3 = int.Parse(Console.ReadLine());
+
+            double multiplication = input1 * input2 * input3;
+
+            Console.WriteLine($"The multiplication of your numbers is: {multiplication}");
+        }
+
+        private static void Task3()
+        {
+            Console.WriteLine("Enter your string");
+            string input = Console.ReadLine();
+            int length = input.Length;
+
+            var result = input[length - 1] + input.Substring(1, length - 2) + input[0];
+
+            Console.WriteLine($"Result of swaping first and last chars is: {result}");
+        }
+
+        private static void Task4()
+        {
+            Console.WriteLine("Pls enter first integer");
+            int num1 = int.Parse(Console.ReadLine());
+            Console.WriteLine("Pls enter second integer");
+            int num2 = int.Parse(Console.ReadLine());
+
+            double result = 0;
+
+            double sum = num1 + num2;
+            Console.WriteLine($"Their sum: {sum}");
+
+            if (num1 == num2)
+            {
+                result = (sum) * 3;
+            }
+            else
+            {
+                result = (sum) / 8;
+            }
+
+            Console.WriteLine($"result of exercise: {result}");
+        }
+
+        private static void Task5()
+        {
+            Console.WriteLine("enter first int");
+            double number1 = int.Parse(Console.ReadLine());
+            Console.WriteLine("enter second int");
+            double number2 = int.Parse(Console.ReadLine());
+            Console.WriteLine("enter third int");
+            double number3 = int.Parse(Console.ReadLine());
+
+            Console.WriteLine($"Largest value = {Math.Max(number1, Math.Max(number2, number3))}");
+            Console.WriteLine($"Lowest value = {Math.Min(number1, Math.Min(number2, number3))}");
+        }
+
+        private static void Task6()
+        {
+            Console.WriteLine("plz input your string");
+            string inputString = Console.ReadLine();
+            Console.WriteLine($"Result of the ex: {StrToUpperCase(inputString)}");
+        }
+
+        public static string StrToUpperCase(string input)
+        {
+            int length = input.Length;
+            int condition = 5;
+
+            if (length == condition)
```

Comment:
requirement : "If a string length is less than 5 characters do nothing." not equal 5